tensorflow/core/kernels/mkl_fused_batch_norm_op.cc

```diff
@@ -821,7 +821,7 @@ class MklFusedBatchNormOp : public OpKernel {
     CHECK_NOTNULL(*saved_mean_tensor);
     // set NAN mean value in case of empty input tensor
```

Comment:
Please also update the comment. Same with other places.

Reply:
@penpornk updated.